src/main/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty.java

```diff
@@ -145,14 +145,17 @@ public PipelineTriggersJobProperty reconfigure(@Nonnull StaplerRequest req, @Che
             throw new Descriptor.FormException(e, null);
         }
 
-        this.stopTriggers();
-
-        PipelineTriggersJobProperty thisProp = new PipelineTriggersJobProperty(new ArrayList<Trigger>(trigList.toList()));
-        thisProp.setOwner(owner);
+        try {
+            owner.removeProperty(this);
```

Comment:
🐛 The property gets removed, but it won't be restored if exception happens at the <code> addTriggersJobPropertyWithoutStart</code> stage